### DISCUSSION TITLE
Changing the git repository

### DISCUSSION THREAD

**Discussion by davidszerman** [Vinzent03/obsidian-git#415] [Q&A] [answered]
Hello,

i configured the git plugin with my vault and it works correctly. 

But I installed gitea on a personal VPS and I would like to use it instead of github.

How could I change that ? 

Thank you for your help

Regards

**Comment by Vinzent03** [comment, marked as answer]
Just change your remote url via the provided commands.

**Reply by davidszerman** [reply]
Hi,

thanks ; is it "initialize a new repo" ?

**Reply by Vinzent03** [reply]
No I think your git repository already exists? If you've previously pushed to GitHub you have to use the edit remotes command.

**Reply by davidszerman** [reply]
it works

thanks
